### ISSUE TITLE
Feature: Playlist support

### ISSUE THREAD

**Issue by BleuLlama** [steffest/BassoonTracker#34]
It would be really great to have a way to enter a url for a playlist/file source.
Perhaps to a specific format json file that could have groups of files listed.
Sort-of the way that it does the hierarchical view of modarchive. 
I've got my favorites at http://umlautllama.com/mods and would be 100% into adding the required file to list out the files for testing, etc.

**Comment by steffest** [comment]
You can already for a single file (if it's served over HTTPS) use the file= parameter.
E.g. https://www.stef.be/bassoontracker/?file=https://modland.com/pub/modules/Fasttracker%202/Reed/a%20synthetic%20device.xm
Support for playlists is a cool idea. I intended Bassoontracker more as a tracker instead of a player, but I must say I use it myself as a player too, so ...
It will probably in the form of a "Jukebox" plugin, where - apart from supplying your own playlist - you can select "genre" or "composer" or "rating" etc.

**Comment by BleuLlama** [comment]
Fair enough, wrt single files.  I'd just like to use it to play through a collection, or share my collection, etc.   I would almost always use protracker, etc to play the files, rarely for editing ;)

**Comment by BleuLlama** [comment]
Hm.  Or "drag a bunch of files in" to create a live playlist.  That would be good as an option as well.

**Comment by blurymind** [comment]
It would be awesome if you use localStorage as a way to bookmark songs directly in the app.

Maybe a playlist can be a type of a bookmark that stores a list of them

**Comment by steffest** [comment]
Excellent idea and easy to implement.

**Comment by steffest** [comment]
@BleuLlama 
There has been some movement on this topic.
I'm currently in the process of implementing playlist support.
We'll see how fancy it turns out to be, but loading external playlists in the most common formats is supported.

For example: to load the "favorites" playlist that is hosted on your site, that would be 
https://www.stef.be/bassoontracker/dev.html?file=https%3A%2F%2Fumlautllama.com%2Fmods%2FPLAYLISTS%2FFavorites.pls

I do filter the playlist to only display .xm and .mod files though, as .IT and .S3M files are not supported yet.
